openapi/components/schemas/Organizations/Organization.yaml

```
@@ -4,10 +4,8 @@ required:
   - country
 properties:
   id:
-    description: The organization identifier string.
-    readOnly: true
     allOf:
-      - $ref: ../ResourceId.yaml
+      - $ref: ../OrganizationId.yaml
   name:
     description: The organization name.
```

nitpick: good to add more detail here
```suggestion
        description: |-
             An organization is an entity that represents you, or your
             company, as a merchant. You can have multiple organizations.
             For more information, see
             [My organizations and websites](https://www.rebilly.com/docs/settings/organizations-and-websites/).
```